pushmi_pullyu.gemspec

```diff
@@ -24,9 +24,13 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.3.1'
 
   spec.add_runtime_dependency 'activesupport', '~> 5.0'
+  spec.add_runtime_dependency 'connection_pool', '~> 2.2'
   spec.add_runtime_dependency 'daemons', '~> 1.2', '>= 1.2.4'
   spec.add_runtime_dependency 'redis', '~> 3.3'
-  spec.add_runtime_dependency 'connection_pool', '~> 2.2'
+  spec.add_runtime_dependency 'rollbar', '~> 2.14'
+
+  # Rollbar recommended json serializer & version
+  spec.add_runtime_dependency 'oj', '~> 2.12.14'
```

Comment:
It's also in their Readme, updated less than 2 months ago https://github.com/rollbar/rollbar-gem/blob/master/README.md, and their Gemfile https://github.com/rollbar/rollbar-gem/blob/master/Gemfile, so I don't think they test against anything beyond that. Safer to stay in the tested zone